projects/dotcms-js/src/lib/core/dotcms-events.service.ts

```diff
@@ -65,7 +66,7 @@ export class DotcmsEventsService {
      *                          messages in the Notification section.
      * @returns any The system events that a client will receive.
      */
-    subscribeTo(clientEventType: string): Observable<DotEventData> {
+    subscribeTo(clientEventType: string): Observable<any> {
```

Comment:
Can we use generics here?

Reply:
No really, I think here can come anything from backend but really I dont still sure about this change